File: Javascript/Mathematics/516.js
/*Assume your brother studies in class 2. 
He has to  complete his homework on co-primes. 
As an elder sibling help him in finding whether the given two numbers is co-prime or not.
Sample Input :
3 5
Sample Output :
1
*/
const readline = require('readline');

const inp = readline.createInterface({
  input: process.stdin
});

inp.on("line", (data) => {
	console.log(coPrime(data))
});
var coPrime=function(n){
  var f=[];
  n=n.split(" ");
  var a=parseInt(n[0]),b=parseInt(n[1]);
 if (a>b){
     var small=b;
 }
 else{
     var small=a;
 }
 for(var i=1;i<=small;i++){
     if ((a%i==0) && (b%i==0)){
         var hcf=i;
     }
     }
     if(hcf==1){
         return 1;
     }
     else
    {
        return 0;
    } 
}

File: Python/Mathematics/516.py
'''Assume your brother studies in class 2.
He has to  complete his homework on co-primes.
As an elder sibling help him in finding whether the given two numbers is co-prime or not.
Sample Input :
3 5
Sample Output :
1'''
a,b=map(int,input().split())
if a>b:
    small=a
else:
    small=b
for i in range(1,small+1):
    if ((a%i==0) and (b%i==0)):
        hcf=i
if hcf==1:
    print("1")
else:
    print("0")
